app/assets/javascripts/discourse/app/instance-initializers/post-decorations.js

```diff
@@ -241,10 +244,10 @@ export default {
       }
 
       api.decorateCookedElement(
-        (post, helper) => {
+        (element, helper) => {
           schedule("afterRender", () => {
-            const tables = post.querySelectorAll(".md-table table");
-            generatePopups(tables, helper.widget.attrs);
+            const tables = element.querySelectorAll(".md-table table");
+            generatePopups(tables, helper.model || helper.widget.attrs);
```

Comment:
Should we make `helper.model` work in the existing widget-based decorations? Then people will be able to switch to it immediately, without having to support both old and new syntaxes.

Reply:
Done